.lando/adminer.sh

```diff
@@ -10,3 +10,23 @@ find . -type f -name 'adminer-*' -print0 | xargs -0 -I{} cp -f {} adminer_librar
 
 chown -R www-data:www-data /var/www/html
 chmod -R +x /var/www/html
+
+#
+# Allow PHP-FPM to read environment variables.
```

Comment:
@tormi I found a way to add env variables.

Reply:
I would add error handling and also use the `enable_clear_env_option()` naming to be more descriptive.